source/database-connections/manage-database-connections.txt

```diff
@@ -0,0 +1,86 @@
+.. _rm-manage-connections:
+
+===========================
+Manage Database Connections
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can copy, edit, or delete database connections from the Relational Migrator
+:guilabel:`Connections` page. Saved connections are listed by their environment
+tag. You can mouse over the :icon-fa5:`info-circle` icon for a connection to
+see a list of projects that use that connection.
+
+Copy a Connection
+-----------------
+
+You can copy an existing connection. This can be useful if you want multiple
+connections to the same server that each connect to different databases, or if
+you have similar connection information for a staging database and a 
+production database.
+
+Copying a connection creates a duplicate with ``-Copy`` appended to the
+connection name.
+
+
+Steps
+~~~~~
+
+To copy a connection:
+
+#. Click the :icon-fa4:`copy` button.
+
+#. Make any desired changes.
+
+#. Click :guilabel:`Save`.
```

Comment:
(copy/paste)
These steps are for editing a connection.

Reply:
I might consider adding just that same explanation. Otherwise it does look a bit like a copy/paste issue.